library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -260,4 +287,29 @@ data class Conversations(
             }
         }
     }
+
+    fun streamAllMessages(): Flow<DecodedMessage> = flow {
+        val topics: MutableList<String> =
+            mutableListOf(
+                Topic.userInvite(client.address).description,
+                Topic.userIntro(client.address).description
+            )
+        for (conversation in list()) {
+            topics.add(conversation.topic)
+        }
+        client.subscribe(topics).collect { envelope ->
+            val conversation = conversations.firstOrNull { it.topic == envelope.contentTopic }
+            if (conversation != null) {
+                val decoded = conversation.decode(envelope)
+                emit(decoded)
+            } else if (envelope.contentTopic.startsWith("/xmtp/0/invite-")) {
+                conversations.add(fromInvite(envelope = envelope))
```

Comment:
Will this update the subscription? I'm not familiar enough with Kotlin flows to understand what happens here. 

Part of the requirements are that when a new conversation is started, the subscriptions will start picking up messages from that conversation.

Reply:
@neekolas in theory yes. However. Flows are pretty hard to test with unit tests. So I'm probably going to have to write this into the example app to really test it.

Reply:
Okay was able to confirm that this works in test. 👍